modules/touch/src/touchinteraction.cpp

```diff
@@ -512,36 +478,10 @@ void TouchInteraction::directControl(const std::vector<TouchInputHolder>& list)
     }
 }
 
-void TouchInteraction::findSelectedNode(const std::vector<TouchInputHolder>& list) {
-    // trim list to only contain visible nodes that make sense
-    // @TODO (emmbr 2023-01-31) This hardcoded list should be removed and replaced by something
-    // else. Either a type of renderable that can always be directly manipulated, or a list
-    // that can be set in config/assets. Or both?
-    std::string selectables[31] = {
-        "Sun", "Mercury", "Venus", "Earth", "Mars", "Ceres", "Jupiter", "Saturn", "Uranus",
```

Comment:
💀 🔫

Reply:
Realized that I forgot to include a "replacement" for this list (that had a second purpose of saying which nodes we can do direct manipulation on). Plan to discuss this during our meeting today